src/content/api/loaders.md

```diff
@@ -25,7 +25,7 @@ Either `return` or `this.callback` can be used to return the transformed `conten
 
 __sync-loader.js__
 
-``` js
+``` javascript
```

Comment:
Why just `js` doesn't work? 🤔

Reply:
we are struggling from mixed declarations, since we use full language name for e.g. `typescript` or `bash` it is preferable to write full name here for consistency so that our *.md files are consistent and maintainable by any user coming from the internet :)